src/Apps/Artwork/Components/ArtworkImageBrowser/ArtworkImageBrowser.tsx

```diff
@@ -46,11 +37,20 @@ export class LargeArtworkImageBrowser extends React.Component<
     // of SSR support in Flickity.
     const carouselImages = typeof window === "undefined" ? [images[0]] : images
 
+    const options = {
+      prevNextButtons: false,
+      wrapAround: true,
+      pageDots: hasMultipleImages,
+      cellAlign: "left",
+      draggable: false,
+      lazyLoad: true,
+    }
```

Comment:
I wonder, wouldn’t you get proper intellisense about available options if you were to define this object inline where the prop is passed to the Carousel component?